amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -2808,6 +2808,29 @@ RpptDataType getRpptDataType(vx_enum vxDataType) {
     }
 }
 
+size_t getDataTypeSize(vx_enum vxDataType) {
```

Comment:
Instead of adding a new function, can we use a map?

We can have a map in internal_rpp.h and directly use it in the copy node